quinn-proto/src/connection/streams/recv.rs

```diff
@@ -287,10 +287,7 @@ impl<'a> Chunks<'a> {
                     self.state = ChunksState::Finished;
                     Ok(None)
                 } else {
-                    let should_transmit =
-                        Self::done(&mut rs, self.read, self.id, self.streams, self.pending);
-                    self.state = ChunksState::Error(ReadError::Blocked, should_transmit);
```

Comment:
Since `Reset` becomes the only possible error kind for `ChunksState::Error` here, should we get rid of the `error` field as part of this commit (or even squash the stuff where the variant gets renamed)?

Reply:
Squashed.